### DISCUSSION TITLE
Requesting assistance with as.Seurat, following Monocle3 Vignette

### DISCUSSION THREAD

**Discussion by BensunCFong** [satijalab/seurat#4883] [Q&A]
Hi there,

Long time lurker and Seurat user, first time seeking help here trying to employ Monocle3 with my data.

**Background**: Generated an integrated Seurat object from two 10X runs (intention differential analysis), and used standard sctransform workflow: anchors, integration, PCA, RunUMAP, FindClusters, FindNeighbors. Then, took a subset of the clusters for object _seurat_integrated_2_, and repeated PCA, RunUMAP, FindClusters, FindNeighbors.

Following the [Monocle3 vignette](http://htmlpreview.github.io/?https://github.com/satijalab/seurat-wrappers/blob/master/docs/monocle3.html) I can use _as.cell_data_set_ on the object to generate and plot a cell_data_set object with `cds <- as.cell_data_set(seurat_integrated_2)`. 

**Issue**: When I attempt to then convert the object back using `integrated.sub <- subset(as.Seurat(cds), monocle3_partitions == 1)`, I recieve the error message: 

```
Error in h(simpleError(msg, call)) : 
  error in evaluating the argument 'x' in selecting a method for function 'subset': One or more of the assays you are trying to convert is not in the SingleCellExperiment object
```

Specifically running `as.Seurat(cds) ` results in the same error. I can't seem to find any evidence elsewhere online of another user seeing the same issue. This might be an incredibly simple solution, any help would be greatly appreciated. I have the latest versions of Monocle3 and SeuratWrappers installed, and have tried with Seurat 4.0.3 as well as the development release.

**Comment by Sa753** [comment]
I have the same issue

**Comment by torkencz** [comment]
Hi, I can take a look at this would you mind giving me an example rds where I can reproduce this and fix it for a future update?
